ontology/aism.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: ontology_detail
+id: aism
+title: Ontology for the Anatomy of the Insect SkeletoMuscular system (AISM)
```

Comment:
@JCGiron I am wondering - is that way to spell SkeletoMuscular really the right one? Seems bad syntax?

Reply:
The uppercase letters are highlighting where the AISM acronym comes from, but it can be changed to "Ontology for the anatomy of the insect skeletomuscular system" if that is more appropriate syntax.

Reply:
In this case, I think its fine.